update_dockerfiles_on_merge.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+REMOTE_UPSTREAM="upstream"
+BRANCH_NAME="automated-branch-for-dockerfiles"
+
+function proceed_to_update() {
+    UPSTREAM=$(cat .git/config | grep -m 1 -B 1 "https://github.com/AdoptOpenJDK/openjdk-docker" | head -n 1 | cut -d '"' -f 2)
+    if [ -z $UPSTREAM ]; then
+        if cat .git/config | grep -Fxq "[remote \"upstream\"]"; then
+            git remote add temp-upstream https://github.com/AdoptOpenJDK/openjdk-docker
+            REMOTE_UPSTREAM="temp-upstream"
+        else
+            git remote add upstream https://github.com/AdoptOpenJDK/openjdk-docker
+        fi
+    else
+        REMOTE_UPSTREAM="$UPSTREAM"
+    fi
+    git checkout master
+    git fetch ${REMOTE_UPSTREAM}
+    git merge ${REMOTE_UPSTREAM}/master
+    local CHECK_LOCAL=$(git branch --list ${BRANCH_NAME})
+    if [[ -z ${CHECK_LOCAL} ]]; then
+        git checkout -b ${BRANCH_NAME}
+    else
+        git branch -D ${BRANCH_NAME}
+        git checkout -b ${BRANCH_NAME}
+    fi
+    source update_all.sh
+    git add 8/ 9/ 10/ 11/ 12/ 13/ 14/ 15/
```

Comment:
This needs to happen based on `supported_versions` available in `common_functions.sh`

Reply:
made changes accordingly